src/shared/user-record-show.c

```diff
@@ -206,6 +206,9 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
         if (hd)
                 printf("   Directory: %s\n", hd);
 
+        if (hr->bulk_directory)
+                printf("   Bulk Dir.: %s\n", hr->bulk_directory);
```

Comment:
there' should probably be an accessor func such as user_record_home_directory() that makes any relative path provided absolute.

Reply:
Discussed during meeting. We're not doing relative paths